src/Components/LeftSideBar.tsx

```diff
@@ -93,14 +94,13 @@ const CurrentLoc = styled.div`
     color: #88869d;
 `;
 
-const LeftSideBar: React.FunctionComponent<Props> = ({ getCurrentLocationWeather, setIsSearchOpen }) => {
+const LeftSideBar: React.FunctionComponent<Props> = ({ setIsSearchOpen, getCurrentLocationWeather, isTempC}) => {
```

Comment:
Let's break the `LeftSideBar` component into multiple components to have more clear functionality.